web/pandas_web.py

```diff
@@ -307,7 +307,10 @@ def roadmap_pdeps(context):
         with open(pathlib.Path(context["target_path"]) / "pdeps.json", "w") as f:
             json.dump(pdeps, f)
 
-        for pdep in sorted(pdeps["items"], key=operator.itemgetter("title")):
+        regex = "(?<=PDEP-)\\d+"
```

Comment:
Shouldn't the regex `r'^PDEP-(\d+)'' work? This seem unnecessarily complicated and feels like it'll return the PDEP part, not the number.